cmd/client-s3.go

```diff
@@ -1263,11 +1263,43 @@ func (c *S3Client) listObjectWrapper(bucket, object string, isRecursive bool, do
 	return c.api.ListObjectsV2(bucket, object, isRecursive, doneCh)
 }
 
-// Stat - send a 'HEAD' on a bucket or object to fetch its metadata.
-func (c *S3Client) Stat(isIncomplete, isFetchMeta, isPreserve bool, sse encrypt.ServerSide) (*ClientContent, *probe.Error) {
+func (c *S3Client) statInComplete(bucket, object string) (*ClientContent, *probe.Error) {
```

Comment:
Actually `statIncomplete` is a misleading name, since stat process is not the incomplete entity here but the file upload is the one.
Can we rename it again? It could be "statIncompleteUploads" or something like that.

Reply:
> Can we rename it again? It could be `"statIncompleteUploads"` or something like that, like existing `"ListIncompleteUploads"`

Done, I called it `statIncompleteUpload`